ui/components/sidebar.tsx

```
@@ -1,49 +1,78 @@
+const MCPIcon = ({ className }: { className?: string }) => (
+	<svg 
+		className={className} 
+		fill="currentColor" 
+		fillRule="evenodd" 
+		height="1em" 
+		style={{flex:'none',lineHeight:1}} 
+		viewBox="0 0 24 24" 
+		width="1em" 
+		xmlns="http://www.w3.org/2000/svg"
+		aria-label="MCP clients icon"
+	>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)